internal/operands/common-templates/resource.go

```diff
@@ -13,11 +13,8 @@ import (
 )
 
 const (
-	GoldenImagesNSname  = "kubevirt-os-images"
-	BundleDir           = "data/common-templates-bundle/"
 	ViewRoleName        = "os-images.kubevirt.io:view"
 	EditClusterRoleName = "os-images.kubevirt.io:edit"
-	Version             = "v0.13.1"
```

Comment:
You will have to update this script https://github.com/kubevirt/common-templates/blob/master/travis_ci/release-templates-ssp-operator.sh, because you changed the location of `Version` variable

Reply:
I will once this PR is merged: https://github.com/kubevirt/common-templates/pull/329